packages/fast-figma-plugin-msft/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@microsoft/fast-figma-plugin-msft",
+  "version": "0.0.0",
+  "description": "A Figma plugin for designing MSFT experiences",
+  "sideEffects": false,
+  "private": true,
+  "scripts": {
+    "build": "webpack --progress --mode=production",
+    "watch": "webpack --progress --watch --mode=production",
+    "prettier": "prettier --config ../../.prettierrc --write \"**/*.{ts,tsx}\"",
+    "prettier:diff": "prettier --config ../../.prettierrc \"**/*.{ts,tsx}\" --list-different",
+    "test": "yarn tslint && yarn build",
+    "tslint": "tslint -c ./tslint.json '**/*.ts{,x}'",
+    "tslint:fix": "tslint -c ./tslint.json --fix '**/*.ts{,x}'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/fast-dna.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Microsoft/fast-dna/issues/new/choose"
+  },
+  "author": {
+    "name": "Microsoft",
+    "url": "https://discord.gg/FcSNfg4"
+  },
+  "license": "MIT",
+  "devDependencies": {
```

Comment:
i think we should add `react` and `react-dom` as peer dependencies

Reply:
This is effectively shipped as an application, not a library so dev-dependency is accurate.